docs/command/atlas-api-teams-getProjectTeam.txt

```diff
@@ -0,0 +1,104 @@
+
+Public Preview: The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
+Admin API capabilities have their own release lifecycle, which you can check via the provided API endpoint documentation link.
+
+
+
+All members of the team share the same project access. To use this resource, the requesting Service Account or API Key must have the Project Read Only role.
+
+This command is autogenerated and corresponds 1:1 with the Atlas API endpoint https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Teams/operation/getProjectTeam.
+
+For more information and examples, see the referenced API documentation linked above.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas api teams getProjectTeam [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --envelope
+     - envelope
+     - false
+     - Flag that indicates whether Application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set envelope=true in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.
+   * - --groupId
+     - string
+     - true
+     - Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.
+       
+       **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
+   * - -h, --help
+     - 
+     - false
+     - help for getProjectTeam
+   * - -o, --output
+     - string
+     - false
+     - preferred api format, can be ["json", go-template] This value defaults to "json".
+   * - --outputFile
+     - string
+     - false
+     - file to write the api output to. This flag is required when the output of an endpoint is binary (ex: gzip) and the command is not piped (ex: atlas command > out.zip)
+   * - --pretty
+     - 
+     - false
+     - Flag that indicates whether the response body should be in the prettyprint format.
+   * - --teamId
+     - string
+     - true
+     - Unique 24-hexadecimal digit string that identifies the team for which you want to get.
+   * - --version
+     - string
+     - false
+     - api version to use when calling the api call [options: "2023-01-01"], defaults to the latest version or the profiles api_version config value if set This value defaults to "2023-01-01".
```

Comment:
```suggestion
     - Version to use when calling the API, formatted as a "YYYY-MM-DD" date string. If you omit this flag, the API uses the version specified in your CLI profile in the ``api_version`` field. If you don't declare ``api_version``, this value defaults to "2023-01-01".
```
One does not "call an API call". The bracketed options aside here also isn't very clear, so let's be more explicit about what it's meant to signal.